src/main/java/org/mitre/synthea/modules/LifecycleModule.java

```diff
@@ -164,7 +164,7 @@ public static void birth(Person person, long time) {
       // should never happen in practice, but can happen in unit tests
       location.assignPoint(person, city);
       person.attributes.put(Person.ZIP, location.getZipCode(city, person.random));
-      attributes.put(Person.BIRTHPLACE, location.randomCityName(person.random));
+      //attributes.put(Person.BIRTHPLACE, location.randomCityName(person.random));
```

Comment:
To account for when we run in "demographics mode" versus "household mode" we may prefer this:
```java
if (!person.attributes.containsKey(Person.BIRTHPLACE) {
  person.attributes.put(Person.BIRTHPLACE, location.randomCityName(person.random));
}
```